packages/qcaml/qcaml.0.1.1/opam

```diff
@@ -0,0 +1,61 @@
+opam-version: "2.0"
+synopsis: "Experimental OCaml library for quantum computing simulation"
+description: """\
+QCaml is a lightweight OCaml library for experimenting with quantum states,
+gates and measurements. It provides tools for learning quantum computing
+concepts and visualizing qubit states on the Bloch sphere.
+
+Features:
+- Single qubit state representation with complex amplitudes
+- Fundamental quantum gates (Hadamard, Pauli-X/Y/Z)
+- Quantum measurements with probabilistic state collapse
+- Interactive Bloch sphere visualization using OpenGL/GLUT
+- Comprehensive test suite"""
+maintainer: "elias.gauthier@etu.u-bordeaux.fr"
+authors: "Elias GAUTHIER"
+license: "Apache-2.0"
+tags: [
+  "quantum"
+  "computing"
+  "simulation"
+  "qubits"
+  "gates"
+  "visualization"
+  "bloch-sphere"
+]
+homepage: "https://github.com/elias-utf8/qcaml"
+doc: "https://elias-utf8.github.io/qcaml/"
+bug-reports: "https://github.com/elias-utf8/qcaml/issues"
+depends: [
+  "ocaml" {>= "5.2"}
+  "dune" {>= "2.9" & >= "2.9"}
+  "dune-configurator"
+  "conf-freeglut"
+  "alcotest" {with-test}
+  "odoc" {with-doc}
+  "bisect_ppx" {with-test}
+]
+build: [
+  ["dune" "subst"] {dev}
+  [
+    "dune"
+    "build"
+    "-p"
+    name
+    "-j"
+    jobs
+    "--promote-install-files=false"
+    "@install"
+    "@runtest" {with-test}
+    "@doc" {with-doc}
+  ]
+  ["dune" "install" "-p" name "--create-install-files" name]
+]
+dev-repo: "git+https://github.com/elias-utf8/qcaml.git"
+url {
+  src: "https://github.com/elias-utf8/qcaml/archive/refs/tags/v0.1.2.tar.gz"
```

Comment:
There's also something with the version.
This now refers to `v0.1.2.tar.gz` yet the opam file is named `packages/qcaml/qcaml.0.1.1/opam`